### PR TITLE
Vega Spec restored when switching to vg from vl in header dropdown.

### PR DESCRIPTION
**Pull request by ganeshpatro321**
Going to vega from vega-lite will open the compiled vega spec. #128

### CODE REVIEW

**Comment by domoritz** [comment]
I'm surprised that you added this to the components. I thought we could easily write this in the action reducers. Do you think you could try that?

File: src/components/header/index.tsx [hunk 1]
```diff
@@ -6,6 +6,7 @@ import Renderer from './renderer';
 
 export function mapStateToProps(state: State, ownProps) {
   return {
+    compiledVegaSpec: state.compiledVegaSpec,
```
Comment (domoritz) [inline src/components/header/index.tsx]: Do we need this?

Comment (ganeshpatro321) [reply src/components/header/index.tsx]: No we do not, updating.

**Comment by ganeshpatro321** [comment]
@domoritz Can you please squash the commits.

**Comment by domoritz** [comment]
Thanks!

**Pull request merged**